app/decorators/base_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class BaseDecorator < SimpleDelegator
```

Comment:
Just thinking - are we using decorators at the moment? What will they do differently to presenters?

Thinking we could just have presenters for now, and not introduct a very similar concept without being really clear why.

What do you think @mseedat-moj ? 🤔

Reply:
I brought it in to mimic Apply and set up any subsequent tickets. The `BaseDecorator` and `BasePresenter` were added to Apply at the same time last year. My understanding is the `Presenter` is designed to output HTML along with general UI based methods, and decorators are 'purer' non-output related classes. I think the distinction is academic and I've used a Presenter in this PR for example rather than a Decorator.

I'm inclined to agree that the `BaseDecorator` in Apply and Review are both redundant.

Reply:
Discussion here: https://mojdt.slack.com/archives/C03JS4V9TPU/p1698839205005959